econml/metalearners.py

```diff
@@ -458,3 +458,8 @@ def _fit_weighted_pipeline(self, model_instance, X, y, sample_weight):
         else:
             last_step_name = model_instance.steps[-1][0]
             model_instance.fit(X, y, **{"{0}__sample_weight".format(last_step_name): sample_weight})
+
+    def shap_values(self, X, *, feature_names=None, treatment_names=None, output_names=None):
```

Comment:
Why is this the only metalearner that has a model_final based shap_values?

Reply:
all the other metalearners they don't have a straightforward final stage to estimate `theta(x)` directly?